<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>QmidiNet - A MIDI Network Gateway via UDP/IP Multicast

   Copyright (C) 2010-2023, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qmidinetOptionsForm</class>
 <widget class="QDialog" name="qmidinetOptionsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qmidinet.qrc">:/images/qmidinet.svg</iconset>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QGroupBox" name="MidiGroupBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>MIDI</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="NumPortsTextLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;Number of Ports:</string>
          </property>
          <property name="buddy">
           <cstring>NumPortsSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="NumPortsSpinBox">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>32</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>8</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="3">
         <layout class="QVBoxLayout">
          <item>
           <widget class="QCheckBox" name="AlsaMidiCheckBox">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&amp;ALSA</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="JackMidiCheckBox">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&amp;JACK</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>8</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="NetworkGroupBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Network</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="InterfaceTextLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;Interface:</string>
          </property>
          <property name="buddy">
           <cstring>InterfaceComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="InterfaceComboBox">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>8</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="UdpAddrTextLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;UDP Address:</string>
          </property>
          <property name="buddy">
           <cstring>UdpAddrComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="UdpAddrComboBox">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="UdpPortTextLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>UDP &amp;Port:</string>
          </property>
          <property name="buddy">
           <cstring>UdpPortSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="UdpPortSpinBox">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>8</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8"/>
 <tabstops>
  <tabstop>NumPortsSpinBox</tabstop>
  <tabstop>AlsaMidiCheckBox</tabstop>
  <tabstop>JackMidiCheckBox</tabstop>
  <tabstop>InterfaceComboBox</tabstop>
  <tabstop>UdpAddrComboBox</tabstop>
  <tabstop>UdpPortSpinBox</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qmidinet.qrc"/>
 </resources>
 <connections/>
</ui>
